attack/scripts/bashrc

```diff
@@ -61,6 +61,44 @@ info() {
 } 1>&2
 readonly -f info
 
+#
+# Starting point.
+#
+
+starting_point() {
+  # Identify the starting point mode.
+  MODE="$(sed '3,3!d' /tasks.yaml)"
+  export MODE
+
+  #Determine the type.
+  if [[ $MODE =~ "internal-instance" ]]; then
+    ssh_internal
+  else
+    if [[ $MODE =~ "node" ]]; then
+      NODE_TYPE="$(sed '4,4!d' /tasks.yaml)"
```

Comment:
As above - prefer `yq` over sed ... I think having `yq` installed in the attack container may prove useful for other reasons in the future too

Reply:
Please, see last comment.